File: src/server/Modules/Organization/Modules.Organization.Core/Entities/Job.cs
using FluentPOS.Shared.Core.Domain;
using FluentPOS.Shared.DTOs.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FluentPOS.Modules.Organization.Core.Entities
{
    public class Job : BaseEntity
    {
        public JobType JobName { get; set; }

        public string Schedule { get; set; }

        public bool Enabled { get; set; }

    }
}

File: src/client/src/app/modules/admin/org/models/Job.ts
import { JobType } from "src/app/core/enums/JobType";

export interface Job {
    id: string;
    createaAt: string;
    updatedAt: string;
    organizationId: string;
    branchId: string;
    jobName: JobType;
    schedule: string;
    enabled: boolean;
    job: string;
}
